packages/experimental/ActivityIndicator/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@fluentui-react-native/experimental-activity-indicator",
+  "version": "0.1.0",
+  "description": "A cross-platform Fluent Activity Indicator component",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
```

Comment:
For my own knowledge, which package did you use to base what info / scripts to copy? I'm curious so I know what to recommend for the future in case some package is missing a script

Reply:
I copied from Shimmer, recommended by the contributing guide.